docs/book/component-guide/toc.md

```diff
@@ -112,8 +112,6 @@
 * [Terraform Modules](../how-to/infrastructure-deployment/stack-deployment/deploy-a-cloud-stack-with-terraform.md)
 * [Register a cloud stack](../how-to/infrastructure-deployment/stack-deployment/register-a-cloud-stack.md)
 * [Infrastructure as code](../how-to/infrastructure-deployment/infrastructure-as-code/README.md)
-  * [Manage your stacks with Terraform](../how-to/infrastructure-deployment/infrastructure-as-code/terraform-stack-management.md)
```

Comment:
Where did we put this one?

Reply:
This just got deleted. i.e. the content from here got pasted / moved into the README.md in the same folder.

Reply:
Nothing links to it, at least internally.